content/advanced/330_servicemesh_using_appmesh/integrate_observability/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Observability"
+date: 2020-01-27T08:30:11-07:00
+weight: 90
+draft: false
+---
+
+ In this chapter, we will dive into key operational metrics and tools such as CloudWatch Container Insights, Cloudwatch Logs and Prometheus that you can leverage within the AWS environment to supercharge your ability to monitor metrics, collect logs, trigger alerts, and trace distributed services.
```

Comment:
'key operational metrics' should maybe be 'key operational signals' or 'data'? As metrics are one of metrics, logs, traces (which you point out later in this sentence), it could be confusing to someone new to the concepts.

Reply:
Updated